### DISCUSSION TITLE
GCP Pub/Sub permission denied

### DISCUSSION THREAD

**Discussion by ayushsearce** [kedacore/keda#2207] [Q&A / Need Help] [answered]
We have used the GCP Pub/Sub trigger and provided the permissions to service account as mentioned in the documentation.
logs provide the error "KEDAScalerFailed 3s (x16 over 7m39s) keda-operator rpc error: code = PermissionDenied desc = Permission monitoring.timeSeries.list denied (or the resource may not exist)."
Even after providing the Owner permissions to the service account it throws same error.
Installed KEDA using "kubectl apply -f https://github.com/kedacore/keda/releases/download/v2.4.0/keda-2.4.0.yaml"
How we can resolve this?

**Comment by tomkerkhove** [comment, marked as answer]
Marking as resolved given this is a duplicate: https://github.com/kedacore/keda/issues/2208
